KachnaOnline.Business/Services/BoardGamesNotifications/NotificationHandlers/MailBoardGamesNotificationHandler.cs

```diff
@@ -0,0 +1,195 @@
+        private async Task SendEmail(string subject, string content, User user)
+        {
+            var config = _smtpOptionsMonitor.CurrentValue;
+            if (string.IsNullOrEmpty(config.Address) || string.IsNullOrEmpty(config.Host) ||
+                string.IsNullOrEmpty(config.Password))
+            {
+                _logger.LogError("SMTP authentication not available.");
+                return;
+            }
+
+            var fromAddress = new MailAddress(config.Address, SmtpConstants.SenderName);
+            var toAddress = new MailAddress(user.Email, user.Name);
+            var smtp = new SmtpClient()
+            {
+                Host = config.Host,
+                Port = config.Port,
+                EnableSsl = true,
+                DeliveryMethod = SmtpDeliveryMethod.Network,
+                UseDefaultCredentials = false,
+                Credentials = new NetworkCredential(config.Address, config.Password)
+            };
+            var message = new MailMessage(fromAddress, toAddress)
+            {
+                Subject = subject,
+                Body = content,
+                IsBodyHtml = true,
+            };
+            await smtp.SendMailAsync(message);
+        }
+
+        /// <inheritdoc />
+        /// <remarks>
+        /// Sends an email to the user who reserved the game notifying about the near expiration and
+        /// the option of extending the reservation.
+        /// </remarks>
+        public async Task PerformReservationItemExpiresSoon(int itemId)
+        {
+            var item = await _boardGamesService.GetReservationItem(itemId);
+            if (item?.ExpiresOn is null)
+            {
+                _logger.LogError("Item expiring soon not found in DB.");
+                return;
+            }
+
+            try
+            {
+                var game = await _boardGamesService.GetBoardGame(item.BoardGameId);
+                var reservation = await _boardGamesService.GetReservation(item.ReservationId);
+                var user = await _userService.GetUser(reservation.MadeById);
+                if (user is null)
+                {
+                    _logger.LogError("User whose reservation is about to expire not found.");
+                    return;
+                }
+
+                var expiration = item.ExpiresOn.Value;
+                // TODO: possibly include a link to frontend card where the game can be extended.
+                var message = $@"Ahoj,<br><br>
+Tvá rezervace na deskovou hru <b>{game.Name}</b> již brzy vyprší, konkrétně 
+<b>{expiration.Day}. {expiration.Month}. {expiration.Year}</b>.
+Domluv se, prosím, s někým ze Studentské Unie na vrácení hry zpět do klubu. V případě, že
+se ti hra zalíbila a rád bys ji měl půjčenou ještě déle, můžeš také požádat v portálu člena
+o prodloužení, nebo ti ji může prodloužit člen Studentské Unie, pokud se s ním domluvíš.<br><br>
+Tvoje KachnaOnline";
```

Comment:
Again, I think the name should be with the space between words.
```suggestion
Tvoje Kachna Online";
```